### DISCUSSION TITLE
Launching Internet TV with OME

### DISCUSSION THREAD

**Discussion by MehdiJafari84** [OvenMediaLabs/OvenMediaEngine#1327] [Help] [answered]
Hello
What is the best solution for launching an Internet TV with 1-5 million viewer using OME?
How many servers with what hardware will I need?
Can OME alone be enough or should I use the dump mechanism and Nginx or Apache?
My servers can be 44 cores with two CPU 2696 V4 and 256G RAM with 20G bandwidth and I would like to present two qualities along with the original quality

If I have one server with above config, how many viewer this server can support?

Thanks in advance for your help

**Comment by getroot** [comment]
Since it is greatly affected by video bitrates, number of streams, encoding quality, etc., we recommend that you measure performance yourself.

We provide a performance measurement tool called OvenRtcTester for WebRTC.
https://airensoft.gitbook.io/ovenmediaengine/performance-tuning
For LLHLS, try using well-known performance measurement tools such as jmeter.

**Reply by MehdiJafari84** [reply]
@getroot Thank you very much for your answer.
There will be only one stream with 4 quality.
I only need LLHLS, Is there any way to simulate high number of viewers?
Do I need Nginx or Apache? Or OEM can work as high performance web server?

**Reply by getroot** [reply, marked as answer]
OME can support throughput of tens of Gbps or more depending on the CPU, but in your case, edge servers configured using a cache server are of course required. This is because even if the video is only 1 Mbps, 1 Tbps of bandwidth is required to transmit to a million people. 5 Tbps to transmit to 5 millon, which is really very high bandwidth. I recommend you use a CDN like CloudFront. In this case, I always use a CDN. There is no need to stress while directly operating Cache Servers with Edge.

As written above, you can directly measure the performance of OME on your server using a popular tool such as Jmeter.

**Reply by MehdiJafari84** [reply]
Thank you for your reply, got it.
Thank you for this great software <3
